src/pages/Company/components/CompanyAside.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import Chip from './Chip';
+
+function CompanyAside({ companyData }) {
+  const { companyTags } = companyData;
+  return (
+    <CompanyTags>
+      <Heading3>태그</Heading3>
+      <ChipsWrap>
+        {companyTags &&
+          companyTags.map((data, idx) => {
```

Comment:
```suggestion
        {companyTags?.map((data, idx) => {
```
***
optional chaining 문법 사용하면 동일한 동작을 조금 더 간결하게 표현할 수 있습니다.
어떻게 동작하는지 공부하고 이해해보시고 적용해보세요~!
***
그리고 data라는 변수명보다는 의미가 잘 드러나는 변수명으로 활용해주세요!